examples/ipython/langevin_integrator_check.ipynb

```diff
@@ -122,7 +122,7 @@
    "outputs": [],
    "source": [
     "for i in range(nframes):\n",
-    "    # generate the next frame (which is sim.nsteps_per_iteration timesteps)\n",
+    "    # generate the next frame (which is sim.n_steps_per_iteration timesteps)\n",
```

Comment:
Weird. I had changed that actually (this is why I could not find `nsteps` anymore, but didnot commit it...